extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/internal/discovery/ManualBridgeJsonParameters.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) 2014,2017 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.smarthome.binding.hue.internal.discovery;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * The {@link ManualBridgeJsonParameters} class defines JSON object, which
+ * contains bridge attributes like IP address. It is used for forced bridge
+ * Discovery.
+ *
+ * @author MAW initial contribution
+ */
+public class ManualBridgeJsonParameters {
+
+    @SerializedName("bridgeid")
```

Comment:
Streamlining the names could imply that the data is coming from the same source, this isn't the case tho, since the BridgeJsonParameters are red from a philips url and the ManualBridgeJsonParameters are red from a bridge directly. The serialization i'm doing is only changing the variables to camel case, so i'd like to stay away from renaming parameters.

Reply:
I see, sounds good then.